karapace/schema_reader.py

```diff
@@ -86,7 +86,8 @@ def parse_avro(schema_str: str):  # pylint: disable=inconsistent-return-statemen
     @staticmethod
     def parse_protobuf(schema_str: str):
         try:
-            return TypedSchema(parse_protobuf_schema_definition(schema_str), SchemaType.PROTOBUF, schema_str)
+            ts = TypedSchema(parse_protobuf_schema_definition(schema_str), SchemaType.PROTOBUF, schema_str)
```

Comment:
why are we storing the result first before returning as opposed to just return the result directly?

Reply:
This code was part of my debugging workaround. Now maybe fixed.